security/index.adoc

```diff
@@ -33,17 +33,13 @@ endif::[]
 [[security-intro-what-are-containers]]
 == What Are Containers?
```

Comment:
It seems OK to me as a fairly quick in-line intro/recap of what containers provide (without bouncing the reader off to another guide), though I'm not 100% against dropping it.